app/services/stats/compute_follow_up_seen_rate.rb

```diff
@@ -0,0 +1,66 @@
+module Stats
+  class ComputeFollowUpSeenRate < BaseService
+    def initialize(follow_ups:, target_delay_days:, consider_orientation_rdv: false)
+      @follow_ups = follow_ups
+      @target_delay_days = target_delay_days
+      @consider_orientation_rdv = consider_orientation_rdv
```

Comment:
```suggestion
      @consider_orientation_rdv_as_start = consider_orientation_rdv_as_start
```
peut-être plus parlant ?